src/api/api.go

```diff
@@ -1464,6 +1465,7 @@ func (a *Api) SendReaction(c *gin.Context) {
 // @Success 204 {string} OK
 // @Failure 400 {object} Error
 // @Param data body Reaction true "Reaction"
+// @Param id path string true "Phone number"
```

Comment:
I am not 100% sure, but I think the parameter name needs to match the parameter name in the annotated URI.

As annotated URI is `// @Router /v1/reactions/{number} [delete]`, I think the line needs to be:

`// @Param number path string true "Phone number"`

At least that's how I understood the swaggo documentation: https://github.com/swaggo/swag?tab=readme-ov-file#use-multiple-path-params. To be honest, I am not sure what happens when one chooses a different name - I guess maybe some interactive stuff breaks in the swagger UI then? Not sure :sweat_smile: But I think it might be better to play it safe here and change the name :) 

Don't worry about the generation of the swagger files, I'll run `swag` after the PR is merged :)

Reply:
Thanks, fixed that up.